build.cmd

```diff
@@ -18,7 +18,9 @@ if %errorlevel% equ 0 (
 )
 
 if "%outloop%" equ "true" (
+        pushd src\System.Private.ServiceModel\tools\setupfiles
 	start /wait BuildWCFTestService.cmd
+        popd
 )
```

Comment:
It looks like there might be a tab/spaces difference here as they aren't lining up in the PR review